tests/test_xlsx2json.py

```diff
@@ -10,19 +11,205 @@
+                "taxId": 9606,
+                "holdDate": "2023-12-31T00:00:00"
+            },
+            "analysis": [
+                {
+                    "analysisTitle": "Variant Detection 1",
+                    "analysisAlias": "VD1",
+                    "description": "An example analysis for demonstration purposes",
+                    "ProjectTitle": "Example Project",
+                    "experimentType": "Whole genome sequencing",
+                    "referenceGenome": "GCA_000001405.27",
+                    "platform": "BGISEQ-500"
+                },
+                {
+                    "analysisTitle": "Variant Detection 2",
+                    "analysisAlias": "VD2",
+                    "description": "An example analysis for demonstration purposes",
+                    "ProjectTitle": "Example Project",
+                    "experimentType": "Whole genome sequencing",
+                    "referenceGenome": "GCA_000001405.27",
+                    "platform": "BGISEQ-500"
+                },
+                {
+                    "analysisTitle": "Variant Detection 3",
+                    "analysisAlias": "VD3",
+                    "description": "An example analysis for demonstration purposes",
+                    "ProjectTitle": "Example Project",
+                    "experimentType": "Whole genome sequencing",
+                    "referenceGenome": "GCA_000001405.27",
+                    "platform": "BGISEQ-500"
+                }
+            ],
+            "sample": [
+                {
+                    "analysisAlias": "VD1",
+                    "sampleInVCF": "sample1",
+                    "bioSampleAccession": "SAME00001"
+                },
+                {
+                    "analysisAlias": "VD2",
+                    "sampleInVCF": "sample1",
+                    "bioSampleAccession": "SAME00001"
+                },
+                {
+                    "analysisAlias": "VD3",
+                    "sampleInVCF": "sample1",
+                    "bioSampleAccession": "SAME00001"
+                },
+                {
+                    "analysisAlias": "VD1",
+                    "sampleInVCF": "sample2",
+                    "bioSampleAccession": "SAME00002"
+                },
+                {
+                    "analysisAlias": "VD2",
+                    "sampleInVCF": "sample2",
+                    "bioSampleAccession": "SAME00002"
+                },
+                {
+                    "analysisAlias": "VD3",
+                    "sampleInVCF": "sample2",
+                    "bioSampleAccession": "SAME00002"
+                },
+                {
+                    "analysisAlias": "VD3",
+                    "sampleInVCF": "sample3",
+                    "bioSampleAccession": "SAME00003"
+                },
+                {
+                    "analysisAlias": "VD4",
+                    "sampleInVCF": "sample4",
+                    "bioSampleObject": {
```

Comment:
That's quite different from what we currently submit to [BioSamples](https://github.com/EBIvariation/eva-submission/blob/7c9fa0c21d81b5fee7fa77dd5e2bb624d43c0770/tests/test_submit_to_bsd.py#L142) but we might iterate over this to improve.